### PR TITLE
[#164595630] set header buttons min 40x40

### PR DESCRIPTION
**Pull request by lmorelli986**
This PR aims to set header buttons size of 40x40 minimum.

Message screen:
Simulator Screen Shot - iPhone X - 2019-03-14 at 18 51 26

============================================================
Android
Schermata 2019-03-14 alle 18 32 46

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🐞 [#164595630](https://www.pivotaltracker.com/story/show/164595630): I pulsanti presenti nell'header devono avere un area attiva attiva minima di 40x40px

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/components/screens/BaseHeader.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ const styles = StyleSheet.create({
 
 interface OwnProps {
   headerTitle?: string;
-  goBack?: () => void;
+  goBack?: (() => void) | boolean;
```
Comment (cloudify) [inline ts/components/screens/BaseHeader.tsx]:
```suggestion
  goBack?: ComponentProps<typeof GoBackButton>["goBack"];
```

File: ts/components/screens/BaseScreenComponent.tsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ interface ContextualHelpProps {
 
 interface OwnProps {
   headerTitle?: string;
-  goBack?: () => void;
+  goBack?: (() => void) | boolean;
```
Comment (cloudify) [inline ts/components/screens/BaseScreenComponent.tsx]:
```suggestion
  goBack?: ComponentProps<typeof BaseHeader>["goBack"];
```

**Comment by cloudify** [comment]
@lmorelli986 this PR does more than changing the size of the buttons!!!

File: ts/components/screens/BaseHeader.tsx (hunk 1 above)
Comment (lmorelli986) [reply ts/components/screens/BaseHeader.tsx]: changed

File: ts/components/screens/BaseScreenComponent.tsx (hunk 2 above)
Comment (lmorelli986) [reply ts/components/screens/BaseScreenComponent.tsx]: changed

**Comment by lmorelli986** [comment]
>@lmorelli986 this PR does more than changing the size of the buttons!!!

Yes it does. Unfortunately I had to let all headers be the same in order to avoid a per-screen style solution

**Pull request merged**